org.gnome.Rhythmbox3.json

```diff
@@ -86,6 +86,59 @@
             ],
             "cleanup": [ "/include" ]
         },
+        /* avahi needs dbus-python */
+        {
+            "name": "dbus-python",
+            "sources": [
+                {
+                    "type": "archive",
+                    "url": "https://pypi.python.org/packages/ad/1b/76adc363212c642cabbf9329457a918308c0b9b5d38ce04d541a67255174/dbus-python-1.2.4.tar.gz#md5=7372a588c83a7232b4e08159bfd48fe5",
+                    "sha256": "e2f1d6871f74fba23652e51d10873e54f71adab0525833c19bad9e99b1b2f9cc"
+                }
+            ],
+            "cleanup": ["/lib", "/share/doc"]
```

Comment:
Is there a way to say, this module is only required for build time and cleanup anything left behind ? As nothing from this seems to be used by avahi (it is required to build avahi client, which we need to get the headers and lib). Or is doing a cleanup like this OK ?

Reply:
`"cleanup": [ "*" ]`

Reply:
Fixed